Gemfile.lock

```diff
@@ -99,7 +99,7 @@ DEPENDENCIES
   rouge
 
 RUBY VERSION
-   ruby 2.6.0p0
+   ruby 2.6.1p33
```

Comment:
This was not intentional. Looks like the version specified in the Gemfile is `2.6.1` so its probably good to keep this in sync.